allgroup.go

```diff
@@ -5,16 +5,15 @@ import (
 	"sync"
 	"time"
 
-	"github.com/aws/aws-sdk-go/service/kinesis"
-	"github.com/aws/aws-sdk-go/service/kinesis/kinesisiface"
+	"github.com/aws/aws-sdk-go-v2/service/kinesis/types"
 )
 
 // NewAllGroup returns an intitialized AllGroup for consuming
 // all shards on a stream
-func NewAllGroup(ksis kinesisiface.KinesisAPI, store Store, streamName string, logger Logger) *AllGroup {
+func NewAllGroup(ksis kinesisClient, store Store, streamName string, logger Logger) *AllGroup {
```

Comment:
it looks like V2 removed the `kinesisiface.KinesisAPI` but I didn't want to give up the ability to mock out the client so I created a local interface for `kinesisClient`, not ideal but should work for time being while they figure out what to do about KinesisAPI

https://github.com/aws/aws-sdk-go-v2/issues/70